students/ChelseaSmith/Lesson4/mailroom.py

```diff
@@ -0,0 +1,84 @@
+# define variables
+record = {"James Smith": [85, 100, 75],  # dictionary with key as donor name and value as list of previous donations
+          "Alex Jones": [25, 25, 25],
+          "Kate Smith": [100],
+          "Kate Jones": [175, 50],
+          "Alex Smith": [200, 200, 200]}
+
+choice = 1  # variable used to choose main menu options
+
+# functions
+
+
+def report(dictin):  # this prints a report given database dictin
+    print(f"Donor name" + " " * 17 + " | Total Given | Num Gifts | Average Gift\n")
+    for k in dictin:
+        sum = 0
+        num = 0
+        for item in dictin[k]:
+            sum = sum + int(item)
+            num = num + 1
+        print(f"{k:<28}${sum:12.2f}{num:10}   ${(sum / num):13.2f}")
+
+
+def getname(dictin):  # ask for name or to list names if user asks to write letter
+    name = input("Please enter the donor's full name or list for a list of donors.")
+    if name == "list":
+        for x in dictin:
+            print(x)
+        name = input("Please enter the donor's full name")
+    return name
+
+
+def checknew(dictin, name):  # returns true if name is not already in list input
+    new = True
+    for x in dictin:
+        if name == x:
+            new = False
+    return new
```

Comment:
you don't need a separate method -> `name in dictin` will give you what you need.